### PR TITLE
feat(requests): requests crud functionality

### PR DESCRIPTION
**Pull request by emukungu**

### CODE REVIEW

File: db/migrations/20190923070722-create_users_table.js [hunk 1]
```diff
@@ -9,15 +9,19 @@ module.exports = {
...
         type: Sequelize.STRING,
         allowNull: false,
         unique: true
       },
+      country: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
       createdAt: {
         allowNull: false,
```
Comment (aldookware) [inline db/migrations/20190923070722-create_users_table.js:26]: can we have a default value here, say timestamp..

**Review by aldookware — CHANGES_REQUESTED** [review]

File: db/migrations/20190923070722-create_users_table.js (hunk 1 above)
Comment (emukungu) [reply db/migrations/20190923070722-create_users_table.js:26]: working on it

Comment (emukungu) [reply db/migrations/20190923070722-create_users_table.js:26]: resolved

**Comment by aldookware** [comment]
@emukungu how do I test this PR?

**Comment by emukungu** [comment]
Let me update the PR

**Review by aldookware — APPROVED** [review]

**Pull request merged**